src/index.js

```diff
@@ -38,7 +38,7 @@ class AuthGoogleXoPlugin {
     xo.registerPassportStrategy(new Strategy(this._conf, async (accessToken, refreshToken, profile, done) => {
       try {
         console.log(profile)
-        done(null, await xo.registerUser('google', profile.displayName))
+        done(null, await xo.registerUser('google', profile.emails[0].value))
```

Comment:
You can simply do `profile.displayName || profile.emails[0].value`, it's not perfect but it should break less setups :)